ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -550,6 +550,23 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.poe.legacy_detect is arista.avd.defined(true) %}
    poe legacy detect
 {%     endif %}
+{%     if ethernet_interface.switchport is arista.avd.defined %}
+{%         if ethernet_interface.switchport.port_security.enable is arista.avd.defined(true) %}
+   switchport port-security
+{%         endif %}
+{%         if ethernet_interface.switchport.port_security.violation.shutdown is arista.avd.defined(true) %}
+   switchport port-security
+{%         elif ethernet_interface.switchport.port_security.violation.protect.enable is arista.avd.defined(true) %}
+   switchport port-security violation protect
+{%         elif ethernet_interface.switchport.port_security.violation.protect.log is arista.avd.defined(true) %}
```

Comment:
I think the log should be added as a nested `if` under the enabled.

Reply:
there are 2 options 
`switchport port-security violation protect`
`switchport port-security violation protect log`

Reply:
Got it, I will update!